src/App.tsx

```diff
@@ -58,6 +58,10 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const theme = (urlParams.get('theme') ||
+		process.env.DESCOPE_FLOW_THEME ||
```

Comment:
Why light and not system?

Reply:
I think we should leave the default value to the SDK (which is indeed light)

so worth fallback to `undefined` 

also let fix the types like we talked today